build.gradle

```diff
@@ -285,7 +285,12 @@ dependencies {
     compile "com.icthh.xm.commons:xm-commons-lep:${xm_commons_version}"
     compile "com.icthh.xm.commons:xm-commons-timeline:${xm_commons_version}"
     compile "com.icthh.xm.commons:xm-commons-metric:${xm_commons_version}"
- 
+
+    compile "com.lowagie:itext:2.1.7"
```

Comment:
Why is it needed?

Reply:
Removed by adding jaspersoft repository. See https://community.jaspersoft.com/questions/1071031/itext-js6-dependency-issue